[Enhancement] turn on graceful exit loop by default (#56002)
StarRocks/starrocks: docs/en/administration/management/BE_configuration.md

- Introduced in: -
-->

##### loop_count_wait_fragments_finish

> - Default:

Reviewer: ```suggestion
- Default: 2
```

Author: done

>   - 2: v3.4+

Reviewer: remove this line

Author: done

>   - 0: < v3.4

Reviewer: remove this line

Author: done

- Type: Int

> - Unit: loops, each loop is 10 seconds

Reviewer: ```suggestion
- Unit: -
```

Author: done

> - Is mutable:

Reviewer: ```suggestion
- Is mutable: Yes
```

Author: done

>   - Yes: v3.4+

Reviewer: remove this line

Author: done

>   - No: < v3.4

Reviewer: remove this line

Author: done

> - Description: The number of loops to be waited when BE/CN process exit. Each loop is a fixed 10 seconds. Set to `0` to disable the loop wait.

Reviewer: ```suggestion
- Description: The number of loops to be waited when the BE/CN process exits. Each loop is a fixed interval of 10 seconds. You can set it to `0` to disable the loop wait. From v3.4 onwards, this item is changed to mutable and its default value is changed from `0` to `2`.
```
